packages/KSegmentedControl/KSegmentedControl.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="segmented-control d-flex" >
+    <KButton
+      v-for="item in options"
+      :key="label(item)"
+      :disabled="disabled(item)"
+      :appearance="appearance(item)"
+      class="justify-content-center"
+      @click="click(value(item))"
```

Comment:
we should use transparent wrappers instead of binding to a single event 
e.g.
https://github.com/Kong/kongponents/blob/master/packages/KButton/KButton.vue#L18